### PR TITLE
Add templates for homework 5 and 6

### PR DESCRIPTION
**Pull request by flandrade**
@jpvillaisaza These are the templates for homework 5 and 6.

### CODE REVIEW

File: homework06/src/Fibonacci.hs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+fibs2 :: [Integer]
+fibs2 = undefined
+
+
+----------------------------------------------------------------------
+-- Exercise 3
+----------------------------------------------------------------------
+
+--streamToList :: Stream a -> [a]
```
Comment (jpvillaisaza) [inline homework06/src/Fibonacci.hs]: @flandrade Is this in comments to wait for the `Stream` implementation? Is it better if we add an empty declaration?

``` haskell
data Stream a
```

Comment (flandrade) [inline homework06/src/Fibonacci.hs]: @jpvillaisaza  Fixed. You're right, it's better now.

**Comment by jpvillaisaza** [comment]
@flandrade Thanks!

**Comment by jpvillaisaza** [comment]
@flandrade By the way, I added you as collaborator so you don't have to keep updating your fork to open pull requests.

**Pull request merged**